README.md

```diff
@@ -1,15 +1,32 @@
 # Aave v3 cross-chain listing template
 
-This template contains an opinionated smart contract template for creating proposals to list assets on the aave polygon v3 pool.
```

Comment:
making the text more generic and provide some specific explanation for the different chains